src/SharedPipelineComponents/IlrSubmissionEvents/SFA.DAS.Provider.Events.Submission/SubmissionEventsProcessor.cs

```diff
@@ -171,6 +181,12 @@ private SubmissionEvent CalculateDelta(IlrDetails currentIlr, IlrDetails lastSee
                 // EPAOrgId is optional in the ilr, so we need to always set it, otherwise if it is null,
                 // the consumer won't know if it hasn't changed or if it's been removed on a subsequent irl submission
                 @event.EPAOrgId = currentIlr.EPAOrgId;
+                @event.GivenNames = currentIlr.GivenNames;
+                @event.FamilyName = currentIlr.FamilyName;
+                @event.CommitmentId = currentIlr.CommitmentId;
+
+                @event.StandardCode = currentIlr.StandardCode;
+                @event.ActualStartDate = currentIlr.ActualStartDate;
```

Comment:
You've added these fields that don't have anything to do with given/family names. Was there a reason?

Reply:
Just saw your commit comment. I'm not sure if these changes will have an impact. I'll speak to Mike about it